app/presenters/data/mock-bill-run.presenter.js

```diff
@@ -0,0 +1,205 @@
+'use strict'
+
+/**
+ * Formats the response for the GET `/data/mock/{bill-run}` endpoint
+ * @module MockBillRunPresenter
+ */
+
+const { convertPenceToPounds, formatAbstractionPeriod, formatLongDate, formatNumberAsMoney } = require('../base.presenter.js')
+
+function go (billingBatch) {
+  const {
+    billingInvoices,
+    billRunNumber,
+    createdAt,
+    fromFinancialYearEnding,
+    netTotal,
+    region,
+    status,
+    scheme,
+    toFinancialYearEnding,
+    batchType: type,
+    transactionFileReference: transactionFile
+  } = billingBatch
+
+  return {
+    dateCreated: formatLongDate(createdAt),
+    status,
+    region: region.name,
+    type,
+    chargeScheme: scheme === 'sroc' ? 'Current' : 'Old',
+    transactionFile,
+    billRunNumber,
+    financialYear: `${fromFinancialYearEnding} to ${toFinancialYearEnding}`,
+    debit: formatNumberAsMoney(convertPenceToPounds(netTotal)),
```

Comment:
We do `formatNumberAsMoney(convertPenceToPounds())` in several places. Do we want to break it out into a function of its own, `_formatPenceAsPounds()` or something?